wave_matching/src/icp.cpp

```diff
@@ -17,11 +17,15 @@ ICPMatcher::ICPMatcher(float res, const std::string &config_path) {
 
     ConfigParser parser;
     double max_corr, t_eps, fit_eps;
-    int max_iter;
+    int max_iter, estimate_method_int;
     parser.addParam("icp.max_corr", &max_corr);
     parser.addParam("icp.max_iter", &max_iter);
     parser.addParam("icp.t_eps", &t_eps);
     parser.addParam("icp.fit_eps", &fit_eps);
+    parser.addParam("icp.lidar_ang_covar", &(this->lidar_ang_covar));
+    parser.addParam("icp.lidar_lin_covar", &(this->lidar_lin_covar));
+    parser.addParam("icp.covar_estimator", &estimate_method_int);
+    this->estimate_method = static_cast<ICPMatcher::covar_method> (estimate_method_int);
```

Comment:
Put this after `parser.load` because the `estimate_method_int` doesn't contain the parameter value until the parameter is loaded?